package.json

```diff
@@ -24,7 +24,7 @@
     "dotenv": "^16.4.5",
     "events-async": "^1.2.1",
     "express": "^4.19.2",
-    "glob": "^10.3.15"
+    "glob": "10.3.14"
```

Comment:
Apparently I fail at reading... was this supposed to be 10.3.4 like the PR/commit title?

Reply:
No, but we could try that. I chose 10.3.14 because 10.3.15 is the one where the ["requires 14.18 or newer" commit](https://github.com/isaacs/node-glob/commit/57f5551d92d8bc05a903bcc2601045e4f5b7fa5a) first appears.

Reply:
I typo'ed the commit message (and the PR title was taken from the commit message). The previous version we shipped with was 10.3.12, so maybe we can try that.

Reply:
The require-a-newer-version-of-14.x commit was the first thing landed after the 10.3.13 release, further suggesting that 10.3.12 is the way to go.

Reply:
https://github.com/nodejs/github-bot/pull/512